admin/src/components/InputText/Index.tsx

```diff
@@ -0,0 +1,26 @@
+import clsx from "clsx";
+import { PropsWithChildren, forwardRef } from "react";
+
+interface InputProps extends PropsWithChildren<JSX.IntrinsicElements["input"]> {
+  className?: string;
+  label: string;
+  error: string;
+}
+export const InputText = forwardRef<HTMLInputElement, InputProps>(
+  ({ label, className, error, ...rest }, ref) => {
+    return (
+      <>
+        <label htmlFor={label} className="flex flex-col">
+          {label}
```

Comment:
not needing to have the same id as label, label can be a ReactNode and be more custom than only a string always